src/components/tutorial.jsx

```diff
@@ -94,34 +94,36 @@ Tutorial.checkIfCompleted = (tutorial, user, preferences) => {
     if (preferences && preferences.preferences && preferences.preferences.tutorials_completed_at && preferences.preferences.tutorials_completed_at[tutorial.id]) {
       return Promise.resolve(preferences.preferences.tutorials_completed_at[tutorial.id]);
```

Comment:
Let's have this static method return the same type consistently in the Promise. Right now if the user does have a stored timestamp for the given tutorial, the promise is returning the datetime string. If not, then it returns false. Let's have this return true, either cast it with a double bang: `Promise.resolve(!!preferences.preferences.tutorials_completed_at[tutorial.id]` or `Promise.resolve(true)`